### PR TITLE
Allow passing empty --worker-tag to worker managers

### PR DESCRIPTION
**Pull request by epicfaace**
- Allow passing empty `--worker-tag`s to worker managers (previously this didn't work). This allows us to run k8s worker managers that are not tied to a particular tag.

Part of https://github.com/codalab/codalab-worksheets/pull/4090, for issue https://github.com/codalab/codalab-worksheets/issues/3975

### CODE REVIEW

File: codalab_service.py [hunk 1]
```diff
@@ -432,7 +432,6 @@ def has_callable_default(self):
         CodalabArg(
             name='worker_manager_{}_tag'.format(worker_manager_type),
             help='Tag of worker for {} jobs'.format(worker_manager_type),
-            default='codalab-{}'.format(worker_manager_type),
```
Comment (epicfaace) [inline codalab_service.py]: undo this change -- we will just pass "" to the env variables

File: codalab/worker/main.py [hunk 2]
```diff
@@ -323,7 +323,7 @@ def main():
         args.checkin_frequency_seconds,
         bundle_service,
         args.shared_file_system,
-        args.tag_exclusive,
+        args.tag_exclusive if args.tag else False,
```
Comment (epicfaace) [inline codalab/worker/main.py]: undo this change and test

**Review by teetone — APPROVED** [review]
